### PR TITLE
update version to 0.5.0

### PR DESCRIPTION
**Pull request by philolo1**

### CODE REVIEW

**Comment by sylvestre** [comment]
I think cargo.lock needs to be updated too, no?

**Comment by philolo1** [comment]
@sylvestre updated.

**Comment by sylvestre** [comment]
i wish your update of cargo.lock didn't update many dependencies in this PR

**Comment by philolo1** [comment]
@sylvestre i just ran "cargo update", is there another way? My guess is that chrono has quite some dependency changes. I don't remember adding any library.

**Comment by tertsdiepraam** [comment]
@philolo1 Just `cargo test` or something like that should fix `Cargo.lock` right? Idk, maybe I'm confused. `cargo update` is specifically for updating all dependencies to the highest versions in `Cargo.toml`

File: Cargo.lock [hunk 1]
```diff
@@ -135,35 +138,35 @@ checksum = "dd8b5dd2ae5ed71462c540258bedcb51965123ad7e7ccf4b9a8cafaa4a63576d"
 
 [[package]]
 name = "parse_datetime"
-version = "0.4.0"
```
Comment (tertsdiepraam) [inline Cargo.lock:138]: Well, this should be in there at least. Is that not changed either?

Comment (tertsdiepraam) [reply Cargo.lock:138]: Oh and the `fuzz/Cargo.lock` should be updated too I guess.

**Comment by cakebaker** [comment]
Maybe it's `cargo update -p <package-name>` you are looking for?

File: Cargo.lock (hunk 1 above)
Comment (philolo1) [reply Cargo.lock:138]: thanks @tertsdiepraam

**Comment by philolo1** [comment]
@cakebaker thanks. 
@tertsdiepraam @sylvestre thanks for the comments, i think i now updated only parse_datetime and fuzz.

**Comment by philolo1** [comment]
@tertsdiepraam @sylvestre any updates, i would like to close the pr's i have here.

**Comment by sylvestre** [comment]
oups, sorry :(

**Comment by sylvestre** [comment]
ok to publish on crates.io ?

**Comment by philolo1** [comment]
Yes

**Comment by sylvestre** [comment]
live: https://crates.io/crates/parse_datetime

**Pull request merged**